demo/site/src/middleware/predefinedPages.ts

```diff
@@ -63,9 +63,9 @@ async function fetchPredefinedPages(domain: string) {
             });
 
             for (const node of paginatedPageTreeNodes.nodes) {
-                if (node.document?.__typename === "PredefinedPage" && node.document.type) {
+                if (node.document?.__typename === "PredefinedPage" && node.document.type === "news") {
                     pages.push({
-                        codePath: `/${language}${predefinedPagePaths[node.document.type]}`,
+                        codePath: `/${language}${predefinedPagePaths.News}`,
```

Comment:
it is true that news is the only predefinedPage type in demo - the code imho still should stay generic - to support other types - as demo will be used as best practice implementation